doc/asciidoc/cloud/index.adoc

```diff
@@ -0,0 +1,28 @@
+
+== Confluent Cloud
+
+Configuring a connection to a Confluent Cloud instance should follow
+link:https://docs.confluent.io/current/cloud/using/config-client.html#java-client[Confluent's Java Client]
+configuration advice, and the advice in <<_kafka_settings, Kafka Settings>> section.
+At a minimum, to configure this, you will need:
+
+* `BOOTSTRAP_SERVER_URL`
+* `API-KEY`
```

Comment:
API_KEY rather than API-KEY?

Reply:
I'll fix it and i'll update the PR. Thanks for pointing that out